css/style.css

```diff
@@ -0,0 +1,104 @@
+._robotoCond {
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+._robotoSlab {
+  font-family: 'Roboto Slab', serif;
+}
+
+._openSansCond {
+  font-family: 'Open Sans Condensed', sans-serif;
+}
+
+._barlowCond {
+  font-family: 'Barlow Condensed', sans-serif;
+}
+
+._barlow {
+  font-family: 'Barlow', sans-serif;
+}
+
+._arial {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+._helvetica {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+._fnt25 {
+  font-size: 25px;
+}
+
+._fnt22 {
+  font-size: 22px;
+}
+
+._fnt18 {
+  font-size: 18px;
+}
+
+._fnt17 {
+  font-size: 18px;
+}
+
+._fnt15 {
+  font-size: 15px;
+}
+
+._fnt14 {
+  font-size: 14px;
+}
+
+._fnt13 {
+  font-size: 13px;
+}
+
+._lspc-5 {
+  letter-spacing: -5px;
+}
+
+._lh1 {
+  line-height: 1;
+}
+
+._sticky {
+  position: sticky;
+  top: 0;
+  z-index: 1000;
+}
+
+._hovRed:hover {
+  color: red;
+}
+
+._pipe::before {
+  content: "|";
+  margin-right: 0.9vw;
+  color: #aaa;
+  font-family: 'Open Sans Condensed', sans-serif;
+}
+
+._pipe2::before {
+  content: "|";
+  margin-right: 0.5em;
+  color: #aaa;
+  font-family: 'Open Sans Condensed', sans-serif;
+}
+
+._searchForm {
+  /* position: absolute; */
+  right: 70px;
+  bottom: 16px;
+  width: 90px;
+  background-image: url("../images/search.png");
+  background-position: right 7px;
+  background-repeat: no-repeat;
+  background-size: 22px 22px;
```

Comment:
- [ ] **Optional** you could use the background [shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/background).

Reply:
Corrected